etherman/etherman.go

```diff
@@ -217,13 +217,19 @@ func (etherMan *Client) WaitTxToBeMined(hash common.Hash, timeout time.Duration)
 
 // EstimateGasSequenceBatches estimates gas for sending batches
 func (etherMan *Client) EstimateGasSequenceBatches(sequences []ethmanTypes.Sequence) (uint64, error) {
+	// Due to issue #1118 we add 20K gas to avoid and OOG
+	// when the gas estimation is processed before a batch consolidation
+	// in the same block
+	const gasLimitIncrease = 20000
+
```

Comment:
Perhaps this could be moved to a config? As not to recompile the code if the value has to be changed.